_includes/head.html

```diff
@@ -8,7 +8,7 @@
 
   {% assign custom_url = site.url | append: site.baseurl %}
   {% assign full_base_url = custom_url | default: site.github.url %}
-  <link rel="stylesheet" href="{{ "/css/main.css" | prepend: full_base_url }}">
+  <link rel="stylesheet" href="{{ "/assets/css/main.css" | prepend: full_base_url }}">
```

Comment:
Huh. Gotcha. It can. That would maintain pre-3.3 compatibility.

@parkr does your implementation ensure the *source* doesn't exist or the *destination*? We definitely want the source checked, so that I can overwrite a theme file, but thinking we also want to check the destination, to prevent invisibly clobbering a site file.